### DISCUSSION TITLE
More information about the new Categories and Filter functionality.

### DISCUSSION THREAD

**Discussion by rowanmoor** [2fast-team/2fast#131] [General]
Hi,

I have seen the new Categories and Filter functionality is showing as a Pro+ function, and I am interested to see what it can do because I think it will be something I want to pay for.  However, at the moment, I can't see any information anywhere about what it does or how it works.  It isn't mentioned in the tutorial yet, and I can't see any links to any pages or screenshots in the Store or on GitHub that show it in action.

Could we have some more information on it, please?  Or a 2 day trial type thing on the Pro+ to check it out, but that may not be possible with the way the Store works.

**Comment by jp-weber** [comment]
Hey,

sorry for the late reply, unfortunately that went completely lost in work. I'm with you that the functions should be included in the tutorial, I too have a trail version in mind, but the implementation is another matter.

I have published a beta version that can be installed in parallel and the functions can be used freely.
https://github.com/2fast-team/2fast/releases/tag/v1.3.0-beta
